server/src/ankaios_server.rs

```diff
@@ -222,13 +229,36 @@ impl AnkaiosServer {
                                     .send(from_server_command)
                                     .await
                                     .unwrap_or_illegal_state();
+                                self.to_agents
```

Comment:
Only an optional improvement. What about writing here a debug trace about sending update_state_success? It would be helpful if you read logs (and you are trying to find out what is going on).

Reply:
I have added a log message.